call-retrain.py

```diff
@@ -3,127 +3,20 @@
 import requests
 
 api_token = os.environ.get("VALOHAI_API_TOKEN")
+project_id = os.environ.get("VH_PROJECT_ID")
```

Comment:
Since these are strictly required,
```suggestion
api_token = os.environ["VALOHAI_API_TOKEN"]
project_id = os.environ["VH_PROJECT_ID"]
```
so you don't end up passing in e.g. `"Token None"` and wondering what the heck is wrong 😁